lib/Spec.AVLTree.fst

```diff
@@ -0,0 +1,592 @@
+module Spec.AVLTree
```

Comment:
Add the copyright headers in all the files, and add yourself as author. See Pulse.Lib.FractionalAnchoredPreorder.fst in lib for an example.

Reply:
comment addressed.